_posts/2019-03-06-release-1.8.0.md

```diff
@@ -0,0 +1,132 @@
+way. We do this by cleaning up and refactoring Flink under the hood to allow
+more efficient feature development in the future. This includes removal of the
+legacy runtime components that were subsumed in the major rework of Flinks
+underlying distributed system
+([FLIP-6](https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=65147077))
+as well as refactorings on the Table API that prepare it for the future
+addition of the Blink enhancements
+([FLINK-11439](https://issues.apache.org/jira/browse/FLINK-11439)).
+
+Nevertheless, this release includes some important new features and bug fixes.
+The most interesting of those we'll highlight below. Please consult the
+[complete changelog](https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12315522&version=12344274)
+or the [release notes](https://ci.apache.org/projects/flink/flink-docs-release-1.8/release-notes/flink-1.8.html)
+for more details.
+
+
+## New Features and Improvements
+
+* **New Support for Schema Migration when restoring Savepoints**: With Flink
+  1.7.0 we added support for changing the schema of state when using the
+  `AvroSerializer`
+  ([FLINK-10605](https://issues.apache.org/jira/browse/FLINK-10605)). With
+  Flink 1.8.0 we made great progress migrating all built-in `TypeSerializers`
+  to a new serializer snapshot abstraction that theoretically allows schema
+  migration. Of the serializers that come with Flink, we now support schema
+  migration for the `PojoSerializer`
+  ([FLINK-11485](https://issues.apache.org/jira/browse/FLINK-11485)) and Java
+  `EnumSerializer`
+  ([FLINK-11334](https://issues.apache.org/jira/browse/FLINK-11334)), as well
+  as for Kryo in limited cases
+  ([FLINK-11323](https://issues.apache.org/jira/browse/FLINK-11323)).
+
+
+* **Continuous cleanup of old state based on TTL
+  ([FLINK-7811](https://issues.apache.org/jira/browse/FLINK-7811))**: We
+  introduced TTL (time-to-live) for Keyed state in Flink 1.6
+  ([FLINK-9510](https://issues.apache.org/jira/browse/FLINK-9510)).  This
+  feature allowed to clean up and make inaccessible keyed state entries when
+  accessing them. In addition state would now also being cleaned up when
+  writing a savepoint/checkpoint.
+
+  Flink 1.8 introduces continous cleanup of old entries for both the RocksDB
+  state backend
+  ([FLINK-10471](https://issues.apache.org/jira/browse/FLINK-10471)) and the heap
+  state backend
+  ([FLINK-10473](https://issues.apache.org/jira/browse/FLINK-10473)). This means
+  that old entries (according to the ttl setting) are continously being cleanup
+  up.
+
+* **New KafkaDeserializationSchema that give direct access to ConsumerRecord
+  ([FLINK-8354](https://issues.apache.org/jira/browse/FLINK-8354))**: For the
+  Flink `KafkaConsumers`, we introduced a new `KafkaDeserializationSchema` that
+  gives direct access to the Kafka `ConsumerRecord`. This now allows access to
+  all data that Kafka provides for a record. This
```

Comment:
```suggestion
  all data that Kafka provides for a record, including the headers. This
```
- I know that people where asking for headers access, so it might be worth mentioning explicitly.

Reply:
For sure!